build.py

```diff
@@ -336,6 +336,8 @@ def do_make(target=None, log=False):
     if modules:
         tmp_mod_dir = tempfile.mkdtemp()
         os.environ['INSTALL_MOD_PATH'] = tmp_mod_dir
+        os.environ['INSTALL_MOD_STRIP'] = 1
```

Comment:
raceback (most recent call last):
  File "/home/khilman/bin/build.py", line 339, in <module>
    os.environ['INSTALL_MOD_STRIP'] = 1
  File "/usr/lib/python2.7/os.py", line 473, in __setitem__
    putenv(key, item)
TypeError: must be string, not int